README.md

```diff
@@ -73,12 +73,17 @@ For more information please see `./CONTRIBUTING.md`.
 This library should always compile with any combination of features (minus `no-std`) on **Rust
 1.41.1** or **Rust 1.47** with `no-std`.
 
-To build with the MSRV you will need to pin some dependencies (also for `no-std`):
+However, if you want to use the library with either of these Rust versions, you will need to pin a
```

Comment:
Either `s/will/may/` or explicitly mention it only applies to `serde` feature.

Reply:
Removed `syn` since its no longer needed.